skimage/feature/fisher_vector.py

```diff
@@ -0,0 +1,203 @@
+"""
+fisher_vector.py - Implementation of the Fisher vector encoding algorithm
+
+This module contains the source code for Fisher vector computation. The
+computation is separated into two distinct steps, which are called separately
+by the user, namely:
+
+learn_gmm: Used to estimate the GMM for all vectors/descriptors computed for
+           all examples in the dataset (e.g. estimated using all the SIFT
+           vectors computed for all images in the dataset, or at least a subset
+           of this).
+
+compute_fisher_vector: Used to compute the Fisher vector representation for a
+                       single set of descriptors/vector (e.g. the SIFT
+                       descriptors for a single image in your dataset, or
+                       perhaps a test image).
+
+Reference: Perronnin, F. and Dance, C. Fisher kernels on Visual Vocabularies
+           for Image Categorization, IEEE Conference on Computer Vision and
+           Pattern Recognition, 2007
+
+Origin Author: Dan Oneata (Author of the original implementation for the Fisher
```

Comment:
What is the license on that code?  We should include it in LICENSE.txt

Reply:
It's the MIT licence. I have spoken with the Dan Oneata and he has given permission to use his original implementation.

Reply:
OK, that's good.  You can see how we've dealt with external BSD licenses here: https://github.com/scikit-image/scikit-image/blob/main/LICENSE.txt#L60

Reply:
@stefanv He granted permission over email, and the original GitHub gist has no associated license. He mentioned in the email that we can consider the code under the MIT license.

What would I need to do in this case? Add the full MIT license to the LICENSE.txt file?

Reply:
Could you please forward me that email?

And then exactly as you say: add the MIT license with his name to the LICENSE.txt file.  You'll see in that file how to specify which filename it applies to.

Reply:
Sent the email.

No problem, I'll add that to the LICENSE.txt file.

Reply:
Done